homeassistant/components/shelly/strings.json

```diff
@@ -120,6 +120,10 @@
     }
   },
   "issues": {
+    "device_not_calibrated": {
+      "title": "Shelly device {device_name} is not calibrated",
+      "description": "Shelly device {device_name} with IP address {ip_address} requires calibration. To calibrate the device, it must be restarted after proper installation on the valve. You can reboot the device in its web panel, go to 'Settings' > 'Device Reboot'."
```

Comment:
The instructions are mixing "restart" and "reboot", should we be consistent?

Reply:
Fixed, 'reboot' is now used.